_data/madewith.json

```
@@ -0,0 +1,62 @@
+[
+    {
+        "name": "Browser UIs",
+        "projects":
+        [
+            {
+                "key": "servoshell",
+                "name": "Servoshell",
+                "link": "https://github.com/servo/servo/tree/main/ports/servoshell",
+                "img": "/img/made-with/servoshell.png",
+                "description": "First-party browser UI built with egui and targeting the testing use case"
```

```suggestion
                "description": "First-party browser UI built with egui"
```

i think we want to avoid further encouraging the belief that Servo is only an engine and not a browser in its own right, given the recent discussion in the TSC about potentially exploring that.